vim25/soap/client.go

```diff
@@ -255,7 +266,11 @@ func (c *Client) RoundTrip(ctx context.Context, reqBody, resBody HasFault) error
 	}
 
 	req.Header.Set(`Content-Type`, `text/xml; charset="utf-8"`)
-	req.Header.Set(`SOAPAction`, `urn:vim25/6.0`)
+	var soapAction bytes.Buffer
+	soapAction.WriteString(c.Ns)
+	soapAction.WriteString("/")
+	soapAction.WriteString(c.Ver)
+	req.Header.Set(`SOAPAction`, soapAction.String())
```

Comment:
No need for bytes.Buffer here, you could do fmt.Sprintf("%s/%s", ...) or even path.Join(c.VimNameSpace, c.VimVersion)